test/library_search/basic.lean

```diff
@@ -152,7 +152,7 @@ attribute [ex] add_lt_add
 
 example {a b c d: nat} (h₁ : a < c) (h₂ : b < d) : max (c + d) (a + b) = (c + d) :=
 begin
-  suggest with ex,
+  -- suggest with ex,
```

Comment:
```suggestion
  suggest with ex,
```
Guessing this was unintentional?

Reply:
```suggestion
```
No, this was actually a stray. I've now deleted it rather than just commenting it out.